src/local/butler/deploy.py

```diff
@@ -218,10 +218,8 @@ def _deploy_zip(bucket_name, zip_path):
 
 def _deploy_manifest(bucket_name, manifest_path):
   """Deploy source manifest to GCS."""
-  if sys.version_info.major == 3:
-    manifest_suffix = '.3'
-  else:
-    manifest_suffix = ''
+  # manifest_suffix refers to majot python version used.
```

Comment:
I would say leave the deploy.py changes for @oliverchang . we probably need to clear up python3= completely and can come in another cl.